test/clj/rems/api/test_over_http.clj

```diff
@@ -0,0 +1,50 @@
+(ns ^:integration rems.api.test-over-http
+  "API tests that use a full HTTP server."
+  (:require [clj-http.client :as http]
+            [clojure.test :refer :all]
+            [rems.api.testing :refer [standalone-fixture]]
+            [rems.config]
+            [rems.db.test-data :as test-data]))
+
+(use-fixtures :once standalone-fixture)
+
+(deftest test-api-sql-timeouts
+  (let [api-key "42"
+        user-id "alice"
+        application-id (test-data/create-application! {:actor user-id})
+        application-id-2 (test-data/create-application! {:actor user-id})
+        save-draft! #(-> (http/post (str (:public-url rems.config/env) "/api/applications/save-draft")
+                                    {:throw-exceptions false
+                                     :as :json
+                                     :headers {"x-rems-api-key" "42"
+                                               "x-rems-user-id" user-id}
+                                     :content-type :json
+                                     :form-params {:application-id application-id
+                                                   :field-values []}})
+                         (select-keys [:body :status]))
+
+        old-handle-command rems.application.commands/handle-command
+        sleep-time (atom nil)
+        slow-handle-command (fn [cmd application injections]
+                              (when-let [time @sleep-time]
+                                (Thread/sleep time))
+                              (old-handle-command cmd application injections))]
+    (rems.email.core/try-send-emails!) ;; remove a bit of clutter from the log
```

Comment:
Could we have some disable for this? If a test does not care about outbox it could skip them all?

Reply:
Could maybe try telling mount not to start the poller

Reply:
I'd add an idea task on the board.

Reply:
#2102